### ISSUE TITLE
This repository is a mess

### ISSUE THREAD

**Issue by derek1906** [fuzzy1450/Fuzzy1450.github.io#1]
There is a thing called "submodules" in git :)

**Comment by fuzzy1450** [comment]
Yeah, I know its awful. Its more for practice than anything really important.

Also who are you and how did you find this?

**Comment by derek1906** [comment]
> [It's like Christmas morning!](//fuzzy1450.github.io)
>
> \- _Sniper_ from TF2

There's a GitHub icon in the lower right corner and I was bored, so I decided why not take a look and that's how I came across this mess.

**Comment by fuzzy1450** [comment]
Right but how'd you find the website? I haven't advertised it anywhere in months.

**Comment by derek1906** [comment]
`toomanysnipers.tf`  is a meme, it's [everywhere](https://www.google.com/search?q=toomanysnipers.tf+site%3Areddit.com) on Reddit.
